### DISCUSSION TITLE
Gluetun + Mullvad shuts down almost immediately

### DISCUSSION THREAD

**Discussion by pau1h** [qdm12/gluetun#1439] [Q&A] [answered]
Hello! I'm trying to set up Gluetun and Mullvad, I followed all of the steps and get the error "vpn: cannot add route for interface: operation not supported: when adding route: {Ifindex: 2 Dst: ::/0 Src: <nil> Gw: <nil> Flags: [] Table: 51820}" I've been trying to fix this for over an hour to no avail, any help would be appreciated. Here is my config: https://pastebin.com/jMHqhmgh and here are my logs: https://pastebin.com/cmXvL7wP. Any help whatsoever would be appreciated, thank you.

**Comment by bnhf** [comment, marked as answer]
@hawkinspaul 

Looking at your log, it appears you may be running an older version of Gluetun -- is that possible? Your healthcheck is using Github.com, and that's been superseded in newer versions by Cloudflare I believe. If you pulled the image from Docker Hub a while back, and are just trying it now, you may need to delete the outdated image first. In Portainer, which I use, there's a slider to force a new image download before deployment.

**Reply by pau1h** [reply]
I feel like an idiot, I had Gluetun installed a couple of years ago and forgot to re-pull the image. Everything works great now. Thank you for your answer!!
Edit: Sidenote, do you know how to make Qbittorrent restart if Gluetun goes down? As of right now, everything breaks if I lose connection for a second.

**Reply by bnhf** [reply]
@hawkinspaul 

There's been a fair amount of discussion about Qbittorrent issues in other threads. I use Deluge myself, which has worked well -- but my recollection is that binding Qbit to tun0 and IPv4 in Qbit(?) has sorted most out. Check out the end of #1277 for better detail.

**Reply by pau1h** [reply]
Much appreciated, thank you!

**Reply by qdm12** [reply]
Thanks @bnhf 🥇

**Comment by bnhf** [comment]
@hawkinspaul 

Also, this header area of the log shows some important stuff about versions and such, so is worth including. I'd suggest posting your config and logs directly here in the discussion -- it's a bit nicer than having to bounce over to Pastebin. Use 3 backticks at the beginning and end of your log, and 3 backticks plus the letters yml at the beginning of your config plus 3 at the end to get nice yaml formatting for your config.

```
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================
Running version latest built on 2022-11-18T09:51:44.899Z (commit 7a3b994)
🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-02-27T15:36:57-07:00 INFO [routing] default route found: interface eth0, gateway 172.19.0.1 and assigned IP 172.19.0.2
2023-02-27T15:36:57-07:00 INFO [routing] local ethernet link found: eth0
2023-02-27T15:36:57-07:00 INFO [routing] local ipnet found: 172.19.0.0/16
2023-02-27T15:36:58-07:00 INFO [firewall] enabling...
2023-02-27T15:36:58-07:00 INFO [firewall] enabled successfully
2023-02-27T15:36:58-07:00 INFO [storage] merging by most recent 13220 hardcoded servers and 13220 servers read from /gluetun/servers.json
2023-02-27T15:36:58-07:00 INFO Alpine version: 3.16.3
2023-02-27T15:36:58-07:00 INFO OpenVPN 2.4 version: 2.4.12
2023-02-27T15:36:58-07:00 INFO OpenVPN 2.5 version: 2.5.6
2023-02-27T15:36:58-07:00 INFO Unbound version: 1.15.0
2023-02-27T15:36:58-07:00 INFO IPtables version: v1.8.8
2023-02-27T15:36:58-07:00 INFO Settings summary:
```
